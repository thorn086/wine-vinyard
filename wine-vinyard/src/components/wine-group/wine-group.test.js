import React from 'react'
import WineGroup from './wine-group'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })
describe('WineGroup component', ()=>{
    const wineGrp = shallow(<WineGroup />)

    it('renders properly',()=>{
        expect(wineGrp).toMatchSnapshot()
    })
})
