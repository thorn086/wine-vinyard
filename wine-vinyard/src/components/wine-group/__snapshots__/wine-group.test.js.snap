// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WineGroup component renders properly 1`] = `ShallowWrapper {}`;
